src/ai/backend/manager/defs.py

```diff
@@ -20,7 +20,9 @@
 
 _RESERVED_VFOLDER_PATTERNS = [r'^\.[a-z0-9]+rc$', r'^\.[a-z0-9]+_profile$']
 RESERVED_DOTFILES = ['.terminfo', '.jupyter', '.ssh', '.ssh/authorized_keys', '.local', '.config']
-RESERVED_VFOLDERS = ['.terminfo', '.jupyter', '.tmux.conf', '.ssh']
+RESERVED_VFOLDERS = ['.terminfo', '.jupyter', '.tmux.conf', '.ssh', 'bin', 'boot', 'dev', 'etc',
+                     'lib', 'lib64', 'media', 'mnt', 'opt', 'proc', 'root', 'run', 'sbin', 'srv',
+                     'sys', 'tmp', 'usr', 'var', 'home']
```

Comment:
* Is this matched against the full path or the name of directory only?
* Could you provide test cases to demonstrate which paths are accepts and not?
  You may add `tests/models/test_vfolder.py`. Include both successful and failing cases.